merkle-tree/concurrent/tests/tests.rs

```diff
@@ -621,6 +593,87 @@ where
     }
 }
 
+/// Checks whether `append_batch` is compatible with equivalent multiple
+/// appends.
+fn compat_batch<H, const HEIGHT: usize>()
+where
+    H: Hasher,
+{
+    const CHANGELOG: usize = 64;
+    const ROOTS: usize = 256;
+
+    let mut rng = thread_rng();
+    let mut seq = 1;
+
+    for batch_size in 2..(1 << HEIGHT) {
```

Comment:
what about batch size 1?

Reply:
This is important to test, since in our program we can just always call append batch and not care how many inputs we actually have.